src/SprykerEco/Zed/Unzer/Business/Import/UnzerPaymentMethodImporter.php

```diff
@@ -61,6 +75,13 @@ public function __construct(
     public function performPaymentMethodsImport(UnzerKeypairTransfer $unzerKeypairTransfer): void
     {
         $paymentMethodsTransfer = $this->unzerPaymentMethodsAdapter->getPaymentMethods($unzerKeypairTransfer);
+        $unzerCredentialsCollectionTransfer = $this->getChildUnzerCredentialsCollectionTransfer($unzerKeypairTransfer);
+
+        if ($unzerCredentialsCollectionTransfer) {
```

Comment:
is seems like `performPaymentMethodsImport` facade method is affected and specification should be adjusted

Reply:
Specification was already updated.